components/TheHeader.vue

```diff
@@ -202,11 +214,29 @@ export default {
       return options.length ? options : null
     },
 
+    getLocaleOptions() {
+      const { $swell } = this
+
+      const options = $swell.locale
+        .list()
+        .map(locale => ({
+          value: locale.code,
+          label: locale.name,
+          icon: locale.icon
+        }))
+      return options.length ? options : null
+    },
+
```

Comment:
@ericingram I'm wondering if we should move this into the `LocaleSelect` component since this format is specific to how it's used there?

Reply:
Fixed with commit 0634d9a.